connaisseur/alert.py

```diff
@@ -151,11 +151,11 @@ def __get_headers(receiver_config):
 def send_alerts(
     admission_request: AdmissionRequest, admit_event: bool, reason: str = None
 ):
+    if not __is_not_hook_image(admission_request):
```

Comment:
maybe we wanna change the not(not(not(not()))) .... so just `if __is_hook_image(): ..` and switch all return values of `__is_not_hook_image`.